workers/data_refinery_workers/processors/salmon.py

```diff
@@ -250,6 +253,10 @@ def _run_salmon(job_context: Dict) -> Dict:
                                        stdout=subprocess.PIPE,
                                        stderr=subprocess.PIPE)
 
+    ## To me, this looks broken: error codes are anything non-zero.
+    ## However, Salmon (seems) to output with negative status codes
+    ## even with succesful executions.
+    ## Possibly related: https://github.com/COMBINE-lab/salmon/issues/55
```

Comment:
We could change this to check for error codes greater than zero.

Reply:
I think it's to leave this as is with the note, because we define a minimal requirement for our desired behavior without worrying about how upstream salmon (heh) handles their non-standard behavior.

Reply:
Hahahaha nice pun.

Sounds good :+1: